src/Common/Entity/ProgramContentResponse.php

```diff
@@ -514,4 +515,20 @@ public function setHomeBanner($home_banner): void
     {
         $this->home_banner = $home_banner;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getLanguageToggler()
+    {
+        return $this->language_toggler;
+    }
+
+    /**
+     * @param mixed $language_toggler
+     */
+    public function setLanguageToggler($language_toggler): void
+    {
+        $this->language_toggler = $language_toggler;
+    }
```

Comment:
returning void here may make this library not useable with some of the older php services; but I personally think that's OK, it will force us to update them.